crates/mem-pool/src/mem_block.rs

```diff
@@ -93,6 +96,7 @@ impl MemBlock {
         let content = MemBlockContent {
             txs: self.txs.clone(),
             withdrawals: self.withdrawals.clone(),
+            new_addresses: self.new_addresses.drain().collect(),
```

Comment:
Nit: use `std::mem::take` to take `txs`, `withdrawals` and `new_addresses`.

Reply:
Fixed